privacy-protections/request-blocking/service-worker.js

```diff
@@ -2,9 +2,13 @@ self.addEventListener('install', (evt) => {
     self.skipWaiting();
 });
 
+self.addEventListener('activate', (evt) => {
+    evt.waitUntil(clients.claim());
```

Comment:
this makes the new service worker start proxying requests right away (and take it over from previous SW that may exist).